h/sql_tasks/tasks/report/create_from_scratch/02_annotations/02_partial_fill.sql

```diff
@@ -0,0 +1,44 @@
+DROP INDEX IF EXISTS report.annotations_uuid_idx;
+DROP INDEX IF EXISTS report.annotations_created_idx;
+DROP INDEX IF EXISTS report.annotations_updated_idx;
+
+INSERT INTO report.annotations (
+    uuid,
+    user_id, group_id, document_id, authority_id,
+    created, updated,
+    deleted, shared, size,
+    parent_uuids, tags
+)
+SELECT
+    annotation.id as uuid,
+    users.id as user_id,
+    groups.id as group_id,
+    document_id,
+    authorities.id as authority_id,
+    annotation.created::date,
+    -- As we do our partial updates based on updated date, it's good if this
+    -- is actually high res timestamp, so we get less overlap
+    annotation.updated,
+    deleted,
+    shared,
+    LENGTH(text) AS size,
+    "references",
+    tags
+-- <<< Here is our fake-out partial run >>>
+FROM (SELECT * FROM annotation LIMIT 500000) AS annotation
```

Comment:
I'm not sure exactly how we want to do this yet, but we can basically up this number. We do need to actually deploy this out at least as far as QA to have a machine that is ready to run it. So it might be a slightly painful procedure. We probably don't want to be too coy about doing it in big jumps depending on what we see.